vinyldns/groups_test.go

```diff
@@ -24,7 +24,7 @@ func TestGroupCreate(t *testing.T) {
 		t.Error(err)
 	}
 	server, client := testTools([]testToolsConfig{
-		testToolsConfig{
```

Comment:
I know it's a bit redundant, but how would you feel about removing `testToolsConfig` from _all_ `*_test.go` files, but in a _separate_ PR? I'm a bit hesitant to make `groups_test.go`'s coding style unique, while the other `*_test.go` files explicitly declare a `testToolsConfig{}`. Please let me know if you disagree.

Reply:
Sure its fine.